config/paths-debian.conf

```diff
@@ -9,6 +9,8 @@ after  = paths-overrides.local
 
 [DEFAULT]
 
+sshd_backed = auto
```

Comment:
typo (`backe[n]d`) here and in the other files, which rightfully caught up by tests on travis.
+ please add a changelog in Enhancements?
otherwise sounds reasonable ;)